packages/twenty-front/src/modules/ui/layout/dropdown/hooks/useIsDropdownOpen.ts

```diff
@@ -0,0 +1,25 @@
+import { DropdownComponentInstanceContext } from '@/ui/layout/dropdown/contexts/DropdownComponentInstanceContext';
+import { isDropdownOpenComponentState } from '@/ui/layout/dropdown/states/isDropdownOpenComponentState';
+
+import { useAvailableComponentInstanceId } from '@/ui/utilities/state/component-state/hooks/useAvailableComponentInstanceId';
+import { useRecoilComponentValueV2 } from '@/ui/utilities/state/component-state/hooks/useRecoilComponentValueV2';
+
+export const useIsDropdownOpen = (
```

Comment:
I don't think we need this hook! useRecoilComponentValueV2 is already the interface we need (you can pass the dropdownComponentInstanceIdFromProps if you want, otherwise it will retrieve it from DropdownComponentInstanceContext by itself)